### ISSUE TITLE
Add links to "outsider" events from a category

### ISSUE THREAD

**Issue by pferreir** [indico/indico#1118]
``` yaml
# Ticket imported from Trac
```

It would be nice to be able to add a link to an event that is located in a different category. This would basically show the event as if it were in the category (maybe with a little graphical indicator?)
This would be like a "soft link'.
We have to take into account what happens when private events are linked from public categories, protection changes, etc...

**Comment by pferreir** [comment]
Just to say this was mentioned on CERN's Mattermost and is still a useful feature.

**Comment by rmatev** [comment]
Something to think about is what happens with links when cloning the target (linked to) event. This can be controlled in the "What should be cloned?" checklist and probably a good default is that all links to the event are cloned in their respective categories.

**Comment by ThiefMaster** [comment]
Good point. I think this would also need to take into account if the user doing the clone is permitted to create events in the category containing the link.

Because IMHO any Indico user should be able to create such a symlink-event in a category where they can create events, regardless of management permissions on the target event as long as that user can access the event.

**Comment by TimSmithCH** [comment]
I see we've missed the 10-year anniversary of this GitHub issue :)
The possibility to soft-link to events (/categories) would be a very useful addition, not only for the use cases above, but also for some outreach use cases we've in mind where we would gather transversally events of the same type from different categories...
